modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -64,6 +64,17 @@
       end
     end
 
+    it 'sets the flashes' do
+      with_okta_user(scopes) do |auth_header|
+        VCR.use_cassette('evss/claims/claims') do
+          post path, params: data, headers: headers.merge(auth_header)
+          token = JSON.parse(response.body)['data']['attributes']['token']
+          aec = ClaimsApi::AutoEstablishedClaim.find(token)
+          expect(aec.flashes).to eq(%w[Hardship POW])
```

Comment:
Same here it looks like
https://github.com/department-of-veterans-affairs/vets-api/pull/5333/files#r525824863